dashboard/config/initializers/backtrace_silencers.rb

```diff
@@ -8,9 +8,15 @@
 
 # In addition to backtrace silencing, we also want to silence annoying deprecations:
 silenced = [
+  # Added in Rails 5.2
   /Single arity template handlers are deprecated/,
   /Dangerous query method \(method whose arguments are used as raw SQL\) called with non-attribute argument\(s\)/,
   /SourceAnnotationExtractor is deprecated! Use Rails::SourceAnnotationExtractor instead/,
```

Comment:
Will these need to be worked on and removed from this silencer list before the upgrade is finished?

Reply:
yep! The way I'm thinking of the remaining upgrade work is basically two sets of checklist items; one is "the remaining test failures" and the other is this list of deprecations.